src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: "Announcing Meshery v0.5.0"
+subtitle: ""
+date: 2021-01-25 08:00:00 +0000
+author: Aisuko Li
+thumbnail: ./meshery.png
+category: Announcements
+tags:
+ - Meshery
+ - Projects
+ - News
+published: true
+redirect_from:
+ - /blog/meshery/announcing-meshery-v0.5.0/
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import operator from "./meshery-operator-dark.svg";
+
+<BlogWrapper>
+
+<p style="color:#3c494f;text-align:center;">Thanks to our open source contributors</p>
+
+We're pleased to announce the realease of Meshery 0.5.0, our first release of 2021! This realease consists of number of significant architectural enhancements and 8 new capabilities are entering alpha.
+
+The 0.5.0 release cycle will good begin for 2021 release cycle, this is one of the most feature dense releases in a while:the Meshery innovation
```

Comment:
The 0.5.0 release starts off the 2021 release cycle with a one of the most...

space after  `:`.